src/SimpleFeatureToggle/FileFeatureLoadingStrategy.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using Newtonsoft.Json;
+using System.Diagnostics;
+
+namespace SimpleFeatureToggle
+{
+    public class FileFeatureLoadingStrategy : IFeatureLoadingStrategy
+    {
+        private readonly IFileReader _fileReader;
+        private string _fileName;
+
+        public FileFeatureLoadingStrategy(string fileName = null, IFileReader fileReader = null)
+        {
+            _fileReader = fileReader ?? new FileReader();
+            _fileName = fileName;
+        }
+
+        public FileFeatureLoadingStrategy From(string fileName)
```

Comment:
I can't see the benefits of being able to change the fileName. Better to just leave it set from the constructor. If you need to read features from another file, then just create a new instance of this strategy. Just trying to make these strategy classes as simple as possible.